src/content/reference/react/useLayoutEffect.md

```diff
@@ -67,6 +67,8 @@ function Tooltip() {
 
 * The code inside `useLayoutEffect` and all state updates scheduled from it **block the browser from repainting the screen.** When used excessively, this makes your app slow. When possible, prefer [`useEffect`.](/reference/react/useEffect)
 
+* If you trigger a render inside `useLayoutEffect`, React will flush all remaining effects including `useEffect`.
```

Comment:
do we use "flush" somewhere else? It sounds a bit to jargon-y.

We also use uppercase words for React specific stuff so we use "Effect". 

```suggestion
* If you trigger a state update inside `useLayoutEffect`, React will execute all remaining Effects immediately including `useEffect`.
```